src/secret_test.js

```diff
@@ -1,69 +1,80 @@
+// @flow
 "use strict";
 
-const fs = require("fs");
+import fs from "fs";
+import {promisify} from "util";
 
-const assert = require("chai").assert;
-const sinon = require("sinon");
-const renderSecret = require("./secret.js");
+import {assert} from "chai";
+import sinon from "sinon";
+import args from "./arguments.js";
+import * as renderSecret from "./secret.js";
 
-describe("secret", () => {
-    beforeEach(() => {
-        sinon
-            .stub(fs, "readFile")
-            .callsFake((filePath, encoding, callback) =>
-                callback(null, "sekret"),
-            );
```

Comment:
Turns out almost all the tests were restoring this mock, so why share it in a `beforeEach`?